sway/commands.c

```diff
@@ -511,14 +512,11 @@ struct cmd_results *cmd_results_new(enum cmd_status status,
 		results->input = NULL;
 	}
 	if (format) {
+		char *error = malloc(256);
 		va_list args;
 		va_start(args, format);
-		size_t length = vsnprintf(NULL, 0, format, args) + 1;
-		char *error = malloc(length);
-		va_end(args);
-		va_start(args, format);
 		if (error) {
-			vsnprintf(error, length, format, args);
+			vsnprintf(error, 256, format, args);
```

Comment:
What's this change all about?

Reply:
Reverted it back to what's in `master`. See the strike-through text in the PR description for the original change.